### DISCUSSION TITLE
樂高增聘軟體工程師達 3 倍！這家高齡 90 歲的玩具公司，會打造出怎樣的元宇宙世界？

### DISCUSSION THREAD

**Discussion by blackline0911** [1101-GS4719-C/Python-Community-GS4719-C#645] [General]
https://buzzorange.com/techorange/2022/05/26/lego-software-engineer/
全球最大積木玩具製造商樂高（Lego）計畫增聘軟體工程師，人數將會是現在的 3 倍，甚至還攜手暢銷遊戲《Fornite》遊戲開發商 Epic Games，打算在一年內推出兒童專屬、家庭友善的元宇宙（metaverse）產品，這家「年近 90」的高齡玩具公司，會如何進擊數位世界？

**Comment by d26488399** [comment]
如果能在虛擬世界裡玩樂高不謹可以改善積木不夠的問題，還可以避免踩到樂高。

**Comment by treeann** [comment]
樓主的分享讓我想起了小時ˋ玩樂高的回憶，並聯想到居然可以在虛擬世界裡玩樂高

**Comment by stan890425** [comment]
希望在遊戲裡面採到樂高會扣血，
這樣未來的小孩才知道我們當年的痛。

**Comment by Yuhsin000** [comment]
這樣是不是會做出有點像minecraft的東西

**Comment by Ben6uo** [comment]
還真的沒有在留意樂高的近況哎 感謝分享

**Comment by liangyinglly** [comment]
樂高這間老公司竟然有想辦法讓自己在這個科技時代活下去，真的值得讚賞。
若樂高公司的工程師管理得當，想必會是一個轉型成功的典範。

**Comment by sys511613** [comment]
文章中也有提到之後會推出電視劇或影集等數位內容
和未來他們推出的元宇宙串聯起來感覺會很有趣

**Comment by leon10456** [comment]
感謝樓主分享

**Comment by 110409538** [comment]
小時候都是用psp玩樂高的遊戲,期待樂高推出更多遊戲.

**Comment by Yao900108** [comment]
這樣也會省下材料費，更加環保，一舉兩得!!

**Comment by HaoFish123** [comment]
感謝分享，元宇宙和樂高感覺很有趣。

**Comment by jcpan821** [comment]
感謝分享~~

**Comment by alexthebigpp** [comment]
元宇宙目前感覺還只是噱頭而已，連fb都做不太出來，樂高可能還早吧
